src/20230926_AaveV3_Ava_EnablingUSDTAsCollateralOnAaveV3AVAXMarket/AaveV3_Avalanche_EnablingUSDTAsCollateralOnAaveV3AVAXMarket_20230926.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3PayloadAvalanche, IEngine, Rates, EngineFlags} from 'aave-helpers/v3-config-engine/AaveV3PayloadAvalanche.sol';
+import {AaveV3Avalanche, AaveV3AvalancheAssets} from 'aave-address-book/AaveV3Avalanche.sol';
+
+/**
+ * @title Enabling USDT as collateral on Aave v3 AVAX Market
+ * @author Marc Zeller - Aave-Chan Initiative
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x5623b5f84f021ad787033b4a1efd9e2de417004d27c5f2e3d7351f9b575574b1
+ * - Discussion: https://governance.aave.com/t/arfc-enabling-usdt-as-collateral-on-aave-v3-avax-market/14632/3
+ */
+contract AaveV3_Avalanche_EnablingUSDTAsCollateralOnAaveV3AVAXMarket_20230926 is
+  AaveV3PayloadAvalanche
+{
+  uint256 public constant BORROW_CAP = 80_000_000; // 80M
+  uint256 public constant SUPPLY_CAP = 100_000_000; // 100M
```

Comment:
Supply cup is wrong, on forum and in readme is 170M

Reply:
markdown has been updated, proposed caps are in line with risk service providers recommendations